hooks/post_gen_project.py

```diff
@@ -1,12 +1,27 @@
 import os
+import shutil
+
+
+def remove_filepath(filepath):
+    if os.path.isfile(filepath):
+        os.remove(filepath)
+    elif os.path.isdir(filepath):
+        shutil.rmtree(filepath)
+
 
 project_name = "{{cookiecutter.project_name}}"
+print(f"project_name: {project_name}")
```

Comment:
Were these prints intentionally included? Or just for debugging (in which case we should remove them)

Reply:
yep a remnant of debugging, will remove